workflow_tests/test_create_content.py

```diff
@@ -0,0 +1,298 @@
+                "template": "blog-post",
+                "template_params": {
+                    "content_length": 1000,
+                    "language": "en-US",
+                    "sections": "[]",
+                    "summary": "a blog post about how important bees are to my honey farm"
+                }
+            }
+        case "one":
+            return {
+                "language": "en-US",
+                "preferences": None,
+                "template": "blog-post",
+                "template_params": {
+                    "content_length": 1000,
+                    "language": "en-US",
+                    "sections": "['Bees and me']",
+                    "summary": "a blog post about how important bees are to my honey farm"
+                }
+            }
+        case "multiple":
+            return {
+                "language": "en-US",
+                "preferences": None,
+                "template": "blog-post",
+                "template_params": {
+                    "content_length": 1000,
+                    "language": "en-US",
+                    "sections": "['Bees and me', 'Honey for you', 'Conclusion']",
+                    "summary": "a blog post about how important bees are to my honey farm"
+                }
+            }
+        case "non_list":
+            return {
+                "language": "en-US",
+                "preferences": None,
+                "template": "blog-post",
+                "template_params": {
+                    "content_length": 1000,
+                    "language": "en-US",
+                    "sections": "1",
+                    "summary": "a blog post about how important bees are to my honey farm"
+                }
+            }
+        case "non_string_elements":
+            return {
+                "language": "en-US",
+                "preferences": None,
+                "template": "blog-post",
+                "template_params": {
+                    "content_length": 1000,
+                    "language": "en-US",
+                    "sections": "[1, 2]",
+                    "summary": "a blog post about how important bees are to my honey farm"
+                }
+            }
+
+def assert_response(create_content_obj, expected):
+    assert create_content_obj.language == expected["language"]
+    assert create_content_obj.preferences == expected["preferences"]
+    assert create_content_obj.template == expected["template"]
+    template_params = create_content_obj.template_params
+    assert template_params.content_length == expected["template_params"]["content_length"]
+    assert template_params.language == expected["template_params"]["language"]
+    assert template_params.sections == expected["template_params"]["sections"]
+    assert template_params.summary == expected["template_params"]["summary"]
+
+@pytest.mark.parametrize("sign_case", sign_cases)
+def test_sign(sign_case):
+    with lilt.ApiClient(configuration) as api_client:
```

Comment:
what benefit is there by doing `with ... as`?

if it is possible to just do `api_client = lilt.ApiClient(configuration)` then we could de-indent everything below